homeassistant/components/meteo_france/weather.py

```diff
@@ -190,7 +205,7 @@ def forecast(self):
                     {
                         ATTR_FORECAST_TIME: self.coordinator.data.timestamp_to_locale_time(
                             forecast["dt"]
-                        ),
+                        ).isoformat(),
```

Comment:
Side note: According to our dev docs the time should be UTC time in isoformat. We don't seem to do any conversion like we do for timestamp sensor entities. This currently returns the local time, I think.

Reply:
Yes, it seems a bit suspicious. Could you confirm @Quentame ?

Reply:
```python
print("-" * 100)
print(f"raw                                (int) : {forecast['dt']}")
print(
    f"timestamp_to_locale_time      (datetime) : {self.coordinator.data.timestamp_to_locale_time(forecast['dt'])}"
)
print(
    f"timestamp_to_locale_time isoformat (str) : {self.coordinator.data.timestamp_to_locale_time(forecast['dt']).isoformat()}"
)
print(
    f"datetime.fromtimestamp        (datetime) : {datetime.fromtimestamp(forecast['dt'])}"
)
print(
    f"datetime.fromtimestamp isoformat   (str) : {datetime.fromtimestamp(forecast['dt']).isoformat()}"
)
print("-" * 100)
```



```shell
----------------------------------------------------------------------------------------------------
raw                                (int) : 1693353600
timestamp_to_locale_time      (datetime) : 2023-08-30 02:00:00+02:00
timestamp_to_locale_time isoformat (str) : 2023-08-30T02:00:00+02:00
datetime.fromtimestamp        (datetime) : 2023-08-30 00:00:00
datetime.fromtimestamp isoformat   (str) : 2023-08-30T00:00:00
----------------------------------------------------------------------------------------------------
```

Beside changing the data type to string, `.isoformat()` isn't changing much (perhaps hourly mode is using `isoformat()`).
Is I understood, we better should use `datetime.fromtimestamp` to be UTC, right ?

Reply:
I don't think `datetime.fromtimestamp` is correct because it seems to return a naïve datetime.
However, I think we should use `homeassistant.util.dt.utc_from_timestamp`, is that correct @MartinHjelmare ?

Reply:
Yes, always use aware times unless there's an api that specifically needs naive times. And always use our dt utils as far as possible.

Reply:
Thanks @MartinHjelmare, let's fix this in a follow-up then.